src/asynchronous/session/style/datagram.rs

```diff
@@ -101,7 +121,7 @@ impl private::SessionStyle for Repliable {
         Self: Sized,
     {
         async {
-            let socket = UdpSocket::bind(format!("127.0.0.1:{}", options.datagram_port)).await?;
+            let socket = UdpSocket::bind(format!("127.0.0.1:{}", 0u16)).await?;
```

Comment:
My mistake, sorry. Please revert this change and keep `datagram_port` in `SessionOptions`. I forgot that the port for datagrams is taken from the config